pkgs/top-level/release.nix

```diff
@@ -166,17 +169,17 @@ let
         };
 
       stdenvBootstrapTools = with lib;
-        genAttrs systemsWithAnySupport
-          (system: {
+        genAttrs (map (p: p.config) platformsWithAnySupport)
+          (config: {
             inherit
               (import ../stdenv/linux/make-bootstrap-tools.nix {
-                localSystem = { inherit system; };
+                localSystem = { inherit config; };
```

Comment:
Hmm, don't want `systemsWithAnySupport` to make a 2-component `config`. I guess either make it a proper `config` to match, or do something a good bit more complex.